command/agent_test.go

```diff
@@ -2820,6 +2820,36 @@ func TestAgent_LogFile_Config(t *testing.T) {
 	assert.Equal(t, 1048576, cfg.LogRotateBytes)
 }
 
+func TestAgent_EnvVar_Overrides(t *testing.T) {
+	configFile := populateTempFile(t, "agent-config.hcl", BasicHclConfig)
+
+	cfg, err := agentConfig.LoadConfigFile(configFile.Name())
+	if err != nil {
+		t.Fatal("Cannot load config to test update/merge", err)
+	}
+
+	assert.Equal(t, false, cfg.Vault.TLSSkipVerify)
+
+	defer os.Unsetenv("VAULT_SKIP_VERIFY")
+
+	os.Setenv("VAULT_SKIP_VERIFY", "true")
```

Comment:
I think you can just use `t.Setenv("VAULT_SKIP_VERIFY", "true")` and it will be unset after the test.

Reply:
Aha! Good catch! I forgot about that since it doesn't work in some of the Agent tests where we need to set environment variables, so I used `os.Setenv` like we did there. I'll refactor to use `t.Setenv` :)